addons/Popochiu/Editor/MainDock/PopochiuGroup/PopochiuGroup.gd

```diff
@@ -150,3 +157,18 @@ func _update_child_count() -> void:
 	if is_instance_valid(_lbl_title):
 		var childs := _list.get_child_count()
 		_lbl_title.text = title + (' (%d)' % childs) if childs > 1 else title
+
+
+func _order_list(node: Node) -> void:
+	node.disconnect('ready', self, '_order_list')
+	
+	# Place the new row in its place alphabetically
+	var place_before: Node = null
+	for row in _list.get_children():
+		if node.name < row.name:
+			place_before = row
+			break
+	
+	if not place_before: return
+	
+	_list.move_child(node, place_before.get_index())
```

Comment:
Unsure what are you doing here, but is this supposed to work only one time? I mean, are you trying to place a single node in a specific position?

Reply:
Yes. I'm moving a node to the position (in the tree) of another one. That pushes the other nodes updating the order in the tree.

Reply:
OK then!